src/common-principles.md

```diff
@@ -248,9 +254,10 @@ recommending a particular naming scheme for including different types of
 source data (such as the raw event logs or parameter files, before conversion to BIDS).
 However, in the case that these data are to be included:
 
-1.  These data MUST be kept in separate `sourcedata` directory with a similar
-    directory structure as presented below for the BIDS-managed data. For example:
-    `sourcedata/sub-01/ses-pre/func/sub-01_ses-pre_task-rest_bold.dicom.tgz` or
+1.  These data MUST be kept in separate `sourcedata` directory.
+    BIDS does not prescribe a particular naming scheme for source data.
+    But it is RECOMMENDED for it to follow BIDS naming convention where possible.
```

Comment:
```suggestion
    BIDS does not prescribe a particular naming scheme for source data,
    but we suggest following BIDS naming conventions to the extent possible.
```

Reply:
per  discussion [above](https://github.com/bids-standard/bids-specification/pull/1741/files#r1568910761)

```suggestion
    BIDS does not prescribe a particular naming scheme for source data.
    But it is recommended for it to follow BIDS naming convention where possible.
```
to be committed along when we converge in suggestions